prow/config.yaml

```diff
@@ -399,6 +399,30 @@ tide:
     - lgtm
     - approved
     - "cncf-cla: yes"
+    - priority/critical-urgent
+    - kind/bug
```

Comment:
does requirement of /sig label get handled by the missingLabels section below?

Reply:
Yes forbidding the `needs-sig` label handles that.